schema/definitions/science.json

```diff
@@ -0,0 +1,29 @@
+{
+    "relativity": {
+        "enum": [
+            "none",
+            "1c-scalar",
+            "2c-quasi",
+            "4c-full"
+        ]
+    },
+    "elements": {
```

Comment:
1. How do we deal with periodic-table.js now, with the list of elements defined here?
2. The current placing in `science.json` seems a bit ad-hoc, maybe we better off isolating this work item (regarding where to put elements) to a future task
3. As we discussed before, "elements" is an overloaded word in web development, I'd suggest spelling "chemicalElementSymbols" is the future